support/rbac.go

```diff
@@ -202,3 +202,37 @@ func CreateUserClusterRoleBinding(t Test, userName string, roleName string) *rba
 
 	return rb
 }
+
+func CreateUserNamespaceRoleBinding(t Test, userName string, namespace *corev1.Namespace, roleName string) *rbacv1.RoleBinding {
```

Comment:
```suggestion
func CreateUserNamespaceClusterRoleBinding(t Test, userName string, namespace *corev1.Namespace, roleName string) *rbacv1.RoleBinding {
```

Reply:
Actually it uses `RoleBinding` instead of `ClusterRoleBinding`, so I changed function name as well.. 
Thank you @sutaakar for reviewing !!